src/Robo/Plugin/Commands/DevelopmentModeCommands.php

```diff
@@ -199,7 +199,11 @@ public function databaseRefreshTugboat(): ResultData
                 ->run();
             $resultData->append($taskResult);
             $this->io()->section("import $siteName database.");
-            $taskResult = $this->taskExec("zcat $dbPath | $dbDriver -h mariadb -u tugboat -ptugboat $dbName")
+            # With `tail +2`, we are temporarily stripping out the first line
+            # of the database dump to remove a sandbox command that MariaDB
+            # does not understand.
+            # See: https://mariadb.org/mariadb-dump-file-compatibility-change/
+            $taskResult = $this->taskExec("zcat $dbPath | tail +2 | $dbDriver -h mariadb -u tugboat -ptugboat $dbName")
```

Comment:
Will this _always_ be there in any database dump or do we need to check for its presence?

Reply:
There will not always be the new sandbox line in the dump file. Currently we only get those from Platform SH. So yes, if this should be universalized, there would have to be a check, something like:

(easier to do in shell rather than wrapping shell with php):
```
if zcat $dbPath | head -1 | grep --silent '^\/\*\!999999'; then
  zcat $dbPath | tail +2 | $dbDriver -h mariadb -u tugboat -ptugboat $dbName
else
  zcat $dbPath | $dbDriver -h mariadb -u tugboat -ptugboat $dbName
fi
```

Reply:
Thank you both for the astute feedback on this! After some discussion on the original ticket, we decided on upgrading `mariadb-client` in Tugboat after all, so we no longer need this PR. I’ll close it after our tooling is updated to no longer reference this branch.